server/routerlicious/packages/lambdas/src/nexus/utils.ts

```diff
@@ -65,3 +65,18 @@ export const isWriter = (scopes: string[], mode: ConnectionMode) =>
 	hasWriteAccess(scopes) && mode === "write";
 
 export const getRoomId = (room: IRoom) => `${room.tenantId}/${room.documentId}`;
+export const getClientRoomId = (clientId: string) => `client#${clientId}`;
+
+export function isSentSignalMessage(obj: any): obj is ISentSignalMessage {
```

Comment:
nit: Would isTargetedSignalMessage be better? Not sure how this feature is named or how it compares to v1

Reply:
see zach's comment [here](https://github.com/microsoft/FluidFramework/pull/19519#discussion_r1517005610). 

I opted for checking for all the properties of v2 signals since in the future we might want to operate on other ISentSignalMessage properties here.